static/resources/yaml/datadog-agent-vanilla.yaml

```diff
@@ -140,30 +146,43 @@ spec:
               value: "yes"
             - name: DOCKER_HOST
               value: unix:///host/var/run/docker.sock
+            - name: DD_CLUSTER_AGENT_ENABLED
+              value: "false"
             - name: DD_LOG_LEVEL
               value: "INFO"
-            - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+            - name: DD_SYSTEM_PROBE_ENABLED
               value: "false"
+            - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+              value: "true"
+            - name: DD_ORCHESTRATOR_CLUSTER_ID
+              valueFrom:
+                configMapKeyRef:
+                  name: datadog-cluster-id
+                  key: id
```

Comment:
This manifest now references a new `ConfigMap` but this `ConfigMap` hasn’t been added to the manifest.
So, the instantiation will most probably fail because the `CM` is missing.